src/core/sentryHandler.ts

```diff
@@ -0,0 +1,72 @@
+import * as Sentry from "@sentry/browser";
+
+import packageJson from "../../package.json";
+import {UserProfileModel} from "../authentication/api/authenticationApiModels";
+import {isDevelopmentEnv, isStaging} from "../utils/globalVariables";
+
+interface SentryExceptionExtraData {
+  title: string;
+  data: any;
+}
+
+type TSentryUserInfo =
+  | UserProfileModel
+  | {
+      username: string;
+    };
+
+const SENTRY_DNS = "WHATEVER_IT_IS";
+const SENTRY_PROJECT_SLUG = "HIPO_PROJECT_TEMPLATE";
+const isDevEnv = isDevelopmentEnv();
+
+function initSentry() {
+  if (!isDevEnv) {
+    const config: Sentry.BrowserOptions = {
+      dsn: SENTRY_DNS,
+      release: `${SENTRY_PROJECT_SLUG}@${packageJson.version}`,
```

Comment:
We should consider adding `.sentryclirc` file or add `sourceMapFilename: "[app-name].js.map"` to `module.exports.output` object. If we don't, we will see errors on minified file. Which is very hard to examine.
For more detail, you can check out: https://docs.sentry.io/cli/configuration/

Reply:
Good point. I will try this configuration on Pascal and then we can add here too